diff --git a/config.def.h b/config.def.h
index 8baac45..bc662aa 100644
--- a/config.def.h
+++ b/config.def.h
@@ -22,9 +22,9 @@ static const char *colors[][3]      = {
 /* tagging */
 static const char *tags[] = { "", "", "", "", "" };
 
-static const char ptagf[] = "[%s %s]";	/* format of a tag label */
-static const char etagf[] = "[%s]";	/* format of an empty tag */
-static const int lcaselbl = 0;		/* 1 means make tag label lowercase */	
+static const char ptagf[] = "%s %s";	/* format of a tag label */
+static const char etagf[] = "%s";	    /* format of an empty tag */
+static const int lcaselbl = 0;		    /* 1 means make tag label lowercase */	
 
 static const Rule rules[] = {
 	/* xprop(1):
