From bbd1631a7b1f5578c30fc699fda4e32ff6ad4b4d Mon Sep 17 00:00:00 2001
From: Samuel Worne <samuel@worne.xyz>
Date: Tue, 10 May 2022 11:06:05 +1000
Subject: [PATCH] Custom dwm tags.

---
 config.def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.def.h b/config.def.h
index 1c0b587..c00cdfc 100644
--- a/config.def.h
+++ b/config.def.h
@@ -19,7 +19,7 @@ static const char *colors[][3]      = {
 };
 
 /* tagging */
-static const char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8", "9" };
+static const char *tags[] = { "1", "2", "3", "4", "5" };
 
 static const Rule rules[] = {
 	/* xprop(1):
-- 
2.33.3

