diff --git a/config.def.h b/config.def.h
index 14f4732..60a15f1 100644
--- a/config.def.h
+++ b/config.def.h
@@ -9,15 +9,15 @@ static const int user_bh            = 32;        /* 0 means that dwm will calcul
 static const char *fonts[]          = { "JetBrains Mono:style:medium:size=10",
 										"Material Icons:size=14" };
 static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+static const char color_navy[]      = "#000f20";
+static const char color_blue[]      = "#294C60";
+static const char color_yellow[]    = "#fdd6b5";
+static const char color_gray[]      = "#E7D8C9";
+static const char color_red[]       = "#fa877f";
 static const char *colors[][3]      = {
-	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+	/*               fg            bg           border   */
+	[SchemeNorm] = { color_gray,   color_navy,  color_navy },
+	[SchemeSel]  = { color_yellow, color_navy,  color_yellow },
 };
 
 /* tagging */
